client/js/.TemplateMap.js

```diff
@@ -1,6 +1,10 @@
 module.exports={
-	 Footer: require('./views/templates/Footer'),
```

Comment:
You shouldn't need to edit this page.

Reply:
Hmm, I think this might have been modified automatically.  I don't remember editing that file.  I see that this file is now requiring the new pages ("About Us", "Our Offerings", etc.), which I don't believe I added myself.  Maybe these things got added automatically?

Reply:
Yep, my bad.